css/styles.css

```
@@ -0,0 +1,210 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Kumbh Sans", sans-serif;
+    font-size: 18px;
+}
+
+
+/* using root and css variables to define all colors from style text and use them later */
+
+:root{
```

```suggestion
:root {
```
Treba popraviti ovo formatiranje - ne sme se lepiti zagrada, jedno prazno mesto izmedju klase i zagrade. Istrazi mogucnost definisanja linter-a/prettier-a